modules/installation-and-upgrade/pages/connect-payg.adoc

```diff
@@ -0,0 +1,117 @@
+[[connect.payg.instances]]
+= Connect {payg} instance
+
+In the three major public cloud providers (AWS, GCP and Azure), SUSE:
+
+* provides customized {payg} product images for SLES, SLES for SAP, etc.
+* operates per-region RMT Servers mirroring repositories for products available as {payg}
+
+This document describes how to connect existing {payg} instance to {productname} server, and gives basic information about instance statuses.
+
+Before using {payg} feature make sure that: 
+
+* {productname} server is deployed in the same region as the {payg} instance
+* valid {scc} credentials are entered in menu:Admin[Setup Wizard > Organization Credentials], even they provide only limited access to products
+
+
+== Connecting {payg} instance
+
+[[proc-conencting-new-payg]]
+.Procedure: Conencting new {payg}  instance
+[role=procedure]
+. In the {productname} {webui}, navigate  to menu:Admin[Setup Wizard > {payg}], and click btn:[Add {payg}].
+. In the new screen, click btn:[Create] to add new {payg} connection data.
+. In the menu:Add new {payg} data[Description], add the description of the newly created connection.
```

Comment:
payg connection is only created after fields are set and user clicks on create button, after the procedure `proc-finishing-payg-connecting`